### ISSUE TITLE
Install Fails

### ISSUE THREAD

**Issue by NicholasJohn16** [EyalAr/lwip#152]
I've been trying to get this to install for awhile now and it's just not working. I've installed all the dependencies listed for `node-gyp` and it's still not working. I'm honestly not sure what the problem is now so posting here out of desperation. I'm on Windows 8.1 x64. Any assistance would be appreciated.

```
gyp ERR! build error
gyp ERR! stack Error: `msbuild` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\build.js:269:23)
gyp ERR! stack     at ChildProcess.emit (events.js:110:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1074:12)
gyp ERR! System Windows_NT 6.3.9600
gyp ERR! command "node" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd c:\Users\Nick\Desktop\tagger\node_modules\lwip
gyp ERR! node -v v0.12.4
gyp ERR! node-gyp -v v1.0.3
gyp ERR! not ok
npm ERR! Windows_NT 6.3.9600
npm ERR! argv "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "lwip" "--save"
npm ERR! node v0.12.4
npm ERR! npm  v2.10.1
npm ERR! code ELIFECYCLE

npm ERR! lwip@0.0.6 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the lwip@0.0.6 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the lwip package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls lwip
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     c:\Users\Nick\Desktop\tagger\npm-debug.log
```

**Comment by landy2014** [comment]
me too，is there anybody helps？？？？？

**Comment by rayliao** [comment]
I have the same problem, under the Windows 7, install the `CSS-Sprite`, lwip rely on, but the lwip installation is not successful.

**Comment by EyalAr** [comment]
@NicholasJohn16 - are you sure you have all of the node-gyp requirements properly installed? Try to install another native module (such as bcrypt) and see if it works.

@landy2014 - Your screenshot clearly says what the problem is.

**Comment by rayliao** [comment]
Trying to install bcrypt, but still won't do

![qq 20150618140824](https://cloud.githubusercontent.com/assets/5601068/8225156/b1d991b4-15c3-11e5-9ffb-ce2a261831d9.jpg)

**Comment by paperscissors** [comment]
Yeah, very odd—I'm on OSX 10.10.5 and I've got all the dependencies installed. Same results on both lwip and bcrypt. I did manage to correct this, however.

I upgraded Xcode and command line tools to Xcode 7; that didn't do it on it's own.

After a bunch of googling I found this comment-issue: (https://github.com/Homebrew/homebrew/issues/40653)

I made the following symlinks:
```
$ cd /usr/local/lib
$ sudo ln -s ../../lib/libSystem.B.dylib libgcc_s.10.5.dylib
$ sudo ln -s ../../lib/libSystem.B.dylib libgcc_s.10.4.dylib
```

and then made sure that /usr/local/lib was in my $PATH, and viola, lwip installed correctly.
